opentech/static_src/src/app/src/components/NoteListingItem/index.js

```diff
@@ -12,10 +14,13 @@ export default class NoteListingItem extends React.Component {
     render() {
         const { user, timestamp, message } = this.props;
         return (
-            <div>
-                <div style={{fontWeight: 'bold'}}>{user} - {timestamp.format('ll')}</div>
-                <div>{message}</div>
-            </div>
+            <li className="note">
+                <p className="note__meta">
+                    <span>{user}</span>
+                    <span className="note__date">{timestamp.format('ll')}</span>
+                </p>
+                <div className="note__content" dangerouslySetInnerHTML={{__html: message}} />
```

Comment:
As part of the rebase i changed this from `<p>` to `<div>` for the dangerous setting of html.